src/averageAges.js

```diff
@@ -37,7 +42,17 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const womenWithChildren = people.filter(woman => withChildren
```

Comment:
variable womenWithChildren is unclear if the additional condition is not specified

Reply:
done